app/assets/javascripts/components/eventNoteType.js

```diff
@@ -0,0 +1,22 @@
+const textMap =  {
+  300: 'SST Meeting',
+  301: 'MTSS Meeting',
+  302: 'Parent conversation',
+  304: 'Something else',
+  305: '9th Grade Experience',
+  306: '10th Grade Experience'
+};
```

Comment:
Handling this may be related to https://github.com/studentinsights/studentinsights/pull/1461 and https://github.com/studentinsights/studentinsights/pull/1481, but I was trying to punt all that from this PR and deal with it next week.